subsys/nrf_security/Kconfig

```diff
@@ -58,6 +58,7 @@ if NRF_SECURITY
 
 config MBEDTLS_PSA_CRYPTO_BUILTIN_KEYS
 	bool
+	default y if SOC_SERIES_NRF92X && (SOC_NRF9280_CPUSEC || SOC_NRF9230_CPUSEC)
```

Comment:
```suggestion
	default y if SOC_SERIES_NRF92X && SOC_NRF9280_CPUSEC
```

Reply:
CPUSEC should not be referenced in this repo at all